client/deposits/utils/index.js

```diff
@@ -158,7 +159,10 @@ export const getDepositScheduleDescriptor = ( {
 		return createInterpolateElement(
 			sprintf(
 				/** translators: %s - deposit schedule, <a> - Settings page URL */
-				__( '%s <a>Change</a>', 'woocommerce-payments' ),
+				__(
+					'%s <a>Change this</a> or <learn_more_href/>',
```

Comment:
Let's use the punctuation, suggested in the issue, in this case add the missing dot at the end:

> Deposits set to XYZ. [Change this](#) or [learn more](#).

Reply:
Fixed.